README.md

```diff
@@ -177,7 +177,7 @@ If the server fails to start (i.e. a port is not specified in the configuration
 3. The server configuration file is malformed and is not in proper JSON format. Refer to the JSON configuration file in this readme for an example of the expected format.
 4. The specified port is out of the allowed port range. The solution is to select a port in the allowable range (0-65535).
 5. The specified port is already in use. The solution is to select another port or terminate the other application using the port.
-6. Security configuration errors. See [Server SSL TLS Support wiki page](https://github.com/ni/grpc-device/wiki/Server-SSL-TLS-Support).
+6. Security configuration errors. See [Server Security Support wiki page](https://github.com/ni/grpc-device/wiki/Server-Security-Support).
```

Comment:
I am sure we would not have made this change. Can you resolve these unintended changes by pulling in the latest changes to your branch? It seems either your branch or the feature branch (target branch) does not have latest changes from the repo.

Reply:
it seems that change is there in `features/ni-dmm` I may have pulled from main, which has that change, but features/ni-dmm not having it earlier. but now it does have